tests/test_stl.cpp

```diff
@@ -237,6 +237,12 @@ TEST_SUBMODULE(stl, m) {
         .def("member_initialized", &opt_exp_holder::member_initialized);
 #endif
 
+#ifdef PYBIND11_HAS_FILESYSTEM
+    // test_fs_path
+    m.attr("has_filesystem") = true;
+    m.def("parent_path", [](std::filesystem::path p) { return p.parent_path(); });
```

Comment:
`const&`? It's a test, so it's not like performance matters...

Reply:
sure